docassemble/ALAutomatedTestingTests/al_set_up_testing.py

```diff
@@ -195,11 +196,12 @@ def push_files( self ):
     """Send files to folders in new branch in github.
     https://pygithub.readthedocs.io/en/latest/examples/Repository.html#create-a-new-file-in-the-repository'''
     # https://pygithub.readthedocs.io/en/latest/github_objects/Repository.html#github.Repository.Repository.create_file"""
+    test_path = 'docassemble/' + self.package_name + '/data/sources/example_test.feature'
+    self.send_file( test_path, 'Add tests/features/example_test.feature', self.example_test_str )  # 2
```

Comment:
Should `example_test.feature` still exist in `tests/features` or should that be moved to `sources` too? (Not urgent, but at some point.)

Reply:
Ah, the commit message needs to be updated. Thank you!